BioinformaticsProject

```diff
@@ -0,0 +1,64 @@
+# Script prepared by Isaiah Murrell-Thomas and Bethany Oceguera 
+# Assume that you are in the bioinformaticsProject directory. This is where you MUST run the script :)
+# Here, you have to assume that the script and files used are in the bioinformaticsProject directory 
+# Usage : bash BioinformaticsProject [muscle tool path] [hmmbuild tool path] [hmmsearch path]
+
+# Example Code to run
+# bash BioinformaticsProject ~/Private/Biocomputing2022/tools/muscle ~/Private/Biocomputing2022/tools/hmmbuild ~/Private/Biocomputing2022/tools/hmmsearch
+
+# CODE START
+
+# Create one file that appends all McrA gene files 
+for A in ./ref_sequences/mcrA*.fasta
+do
+cat $A >> allspecies.fasta
+done
+
+# Using the msucle tool,, align the allspecies.fasts file to evaluate the level of sequence conservation or homology amongst sequences
+$1 -in allspecies.fasta -out allspecies.fasta.aligned
+
+# Build a profile HMM using the allspecies.fasta sequence alignment and hmmbuild
+$2 allspecies.fasta.hmmbuild allspecies.fasta.aligned
+
+# Search the proteome for McrA gene using hmmsearch
+for B in ~/Private/Biocomputing2022/bioinformaticsProject/proteomes/proteom*.fasta
+do
+$3 --tblout $B.hmmsearch.mcrA allspecies.fasta.hmmbuild $B
+done
+# Create one file that appends all hsp gene files
+for D in ./ref_sequences/hsp70gene_*.fasta
+do
+cat $D >> allgenes.fasta
+done
+
+# Using muscle, align to evaluate the level of sequence conservation or homology amongst sequences
+$1 -in allgenes.fasta -out allgenes.fasta.aligned
+
+# Build a profile HMM using the allspecies.fasta sequence alignment and hmmbuild
+$2 allgenes.fasta.hmmbuild allgenes.fasta.aligned
+
+# Search the proteome for hsp gene using hmmsearch
+for E in ./proteomes/proteom*.fasta
+do
+$3 --tblout $E.hmmsearch.hsp allgenes.fasta.hmmbuild $E
+done
+
+# Creating a table listing the proteomes #, if it is a methanogen (0=no, 1=yes), and how many hsp that proteome has. This table is sent to standard out
+echo "Preteome mcrA HSP70" > SummaryTable
+
+for F in ./proteomes/proteom*.fasta
+do
+proteome=$(echo $F | sed -E 's/[^0-9]//g')
+mcrA=$(cat $F.hmmsearch.mcrA | grep -v -E "^#" | wc -l)
+HSP70=$(cat $F.hmmsearch.hsp | grep -v -E "^#" | wc -l)
+echo $proteome $mcrA $HSP70 >> SummaryTable
+done
+
+cat SummaryTable
+
+# Create file of candidate pH-resistant methanogens
+# run the code "cat candiadtes" to see all the genes that have at least one mcrA gene and at least one HSP70 gene
+echo "Eligible pH-resistant Candidates" > candidates
+cat SummaryTable | grep -v -w "0" | cut -d " " -f 1 | sed -E "s/^/Proteome /g" | tail -n +2 >> candidates
+
+# CODE END
```

Comment:
-header comments, including usage line [1 point]
	-general commenting of code throughout script [1 point]
	-code efficiency [1 points] 
=> 1 point was taken off since you don't need the for loop for lines 11-15 and 28-32
You can also make a for loop for hmm search for both Hsp and mcrA, as well as outputting the table form in just one for loop.